#!/bin/sh /usr/share/dpatch/dpatch-run

## 03_cdbremse
## http://vdr-portal.de/board/thread.php?postid=429254#post429254
## All lines beginning with `## DP:' are a description of the patch.
diff -Nur cdda-0.1.0.orig/cdda_device.c cdda-0.1.0/cdda_device.c
--- cdda-0.1.0.orig/cdda_device.c	2005-12-23 10:10:30.000000000 +0100
+++ cdda-0.1.0/cdda_device.c	2006-03-06 21:59:25.000000000 +0100
@@ -228,6 +228,7 @@
       mp_cdIo = cdio_open(mp_device, DRIVER_LINUX);
       if (NULL == mp_cdIo)
         return scCannotOpenDevice;
+	cdio_set_speed(mp_cdIo,2);
     } else
       return scOutOfMemory;
   } else
